src/api/routers/search.py

```
@@ -0,0 +1,390 @@
+        )
+
+@router.get(
+    "/similar/{chunk_id}",
+    response_model=SearchResponse,
+    summary="Find similar content to a specific chunk",
+    description="Find content similar to a specific chunk by its ID."
+)
+async def find_similar_content(
+    chunk_id: str,
+    k: int = Query(10, description="Number of similar results to return", ge=1, le=100),
+    content_types: Optional[List[ContentType]] = Query(None, description="Filter by content types"),
```

Ruff (0.13.1)

Do not perform function call `Query` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)